projects/imminence/docker-compose.yml

```diff
@@ -17,28 +18,39 @@ x-imminence: &imminence
   working_dir: /govuk/imminence
 
 services:
+  imminence-pg:
+    image: ghcr.io/baosystems/postgis:14-3.2
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_PASSWORD: pgsecret
```

Comment:
Do we need a password? I think we don't for our existing postgres container.

Reply:
We needed it for the old container, but if we're okay with the new one, it's unnecessary.